README.md

```diff
@@ -12,7 +12,7 @@ Is your software so tightly coupled to your CPU that unit testing seems impossib
 
 The Bareflank **P**rocessor **A**bstraction **L**ayer transforms facts about your CPU into a software support library. This lets you access and manipulate the low-level details of your hardware through a convinient software API. Supose you are working on a C++ project for the Intel platform, and compiling/assembling with a GNU toolchain:
 
-0. Install Python 3
+0. Install Python 3 and pip install lxml
```

Comment:
Nice catch, I think it might need to be even more specific:

Install Python > 3.6 and ```pip install lxml dataclasses colorama```

Out of curiosity, which version of python are you using? Can you test with installing those other two packages?

Reply:
Im using 3.7.4. Using those packages works fine too